stingray/io.py

```diff
@@ -653,3 +654,31 @@ def get_btis(gtis, start_time=None, stop_time=None):
 def gti_len(gti):
     """Return the total good time from a list of GTIs."""
     return np.sum([g[1] - g[0] for g in gti])
+
+
+def save_object(object, filename):
+    """
+    Pickle a class instance.
+
+    Parameters
+    ----------
+    object: a class instance
+    filename: str (name of the file to be created)
+
+    """
+
+    filename = "stingray/data/" + filename
```

Comment:
I think both of them are similar. I have seen libraries creating the file in the current location only e.g. NetworkX dumps Graph data, scikit-image saves the image. I have no problem with saving it into the data directory or otherwise.

PS. If the user runs 
```python
>>> save_object(obj, "data/myobject.file")
```
I think that should create the data directory. I'm not sure though.